it/v2/connectors/ValidationConnectorSpec.scala

```diff
@@ -322,6 +326,92 @@ class ValidationConnectorSpec
 
     }
 
+    "When validating large messages" - {
+
+      "then on successful validation of schema valid XML, must return NO_CONTENT" in {
+        server.stubFor(
+          post(
+            urlEqualTo("/transit-movements-validator/messages/IE015/validation")
+          )
+            .withHeader(XObjectStoreUriHeader, equalTo("resource/location"))
+            .willReturn(
+              aResponse().withStatus(NO_CONTENT)
+            )
+        )
```

Comment:
No need to pass extra headers. Please remove it for all down wards test cases as well

Reply:
These test case will fail if the response is removed!

Reply:
just check this part. don't think so we need to pass HeaderNames.ACCEPT -> ContentTypes.JSON? implicit val hc: HeaderCarrier = HeaderCarrier(extraHeaders = Seq(HeaderNames.ACCEPT -> ContentTypes.JSON))